zowex/src/main.rs

```diff
@@ -229,7 +227,7 @@ fn exit_when_alt_cmd_needed(cmd_line_args: &Vec<String>) {
  * @returns
  *      A String containing all of the command line arguments.
  */
-fn arg_vec_to_string(arg_vec: &Vec<String>) -> String {
+fn arg_vec_to_string(arg_vec: &[String]) -> String {
```

Comment:
Since this function no longer takes a Vec type of parameter, should we change the name of the function?

Reply:
The name for the Rust datatype `&[String]` seems to be "slice" instead of "vector".

I'm not sure what to rename the function to. I don't think `arg_array_to_string` or `arg_list_to_string` would be technically accurate. And `arg_slice_to_string` or `arg_collection_to_string` might be, but they sound kind of weird IMO.

If there is a name that stands out to you as a good option, I'm happy to change it. Otherwise I'm inclined to leave it as is, since we're still passing a `Vec` parameter into these methods.

Reply:
I agree that we can leave it as it is.